core/storage/question/gae_models_test.py

```diff
@@ -492,6 +536,49 @@ def test_get_question_skill_links_equidistributed_by_skill(
         question_ids = [link.question_id for link in question_skill_links]
         self.assertEqual(question_ids.count('question_id2'), 1)
 
+    def test_get_random_question_skill_links_equidistributed_by_skill(self):
+        questionskilllink_model1 = (
+            question_models.QuestionSkillLinkModel.create(
+                'question_id1', 'skill_id1', 0.1)
+            )
+        questionskilllink_model2 = (
+            question_models.QuestionSkillLinkModel.create(
+                'question_id2', 'skill_id1', 0.5)
+            )
+        questionskilllink_model3 = (
+            question_models.QuestionSkillLinkModel.create(
+                'question_id3', 'skill_id1', 0.8)
+            )
+        questionskilllink_model4 = (
+            question_models.QuestionSkillLinkModel.create(
+                'question_id4', 'skill_id1', 0.9)
+            )
+        questionskilllink_model5 = (
+            question_models.QuestionSkillLinkModel.create(
+                'question_id5', 'skill_id1', 0.6)
+            )
+        question_models.QuestionSkillLinkModel.put_multi_question_skill_links(
+            [questionskilllink_model1, questionskilllink_model2,
+             questionskilllink_model3, questionskilllink_model4,
+             questionskilllink_model5])
+        utils.random.seed(1)
+        question_skill_links_1 = (
+            question_models.QuestionSkillLinkModel.
+            get_question_skill_links_equidistributed_by_skill(
+                3, ['skill_id1']
+            )
+        )
+        utils.random.seed(3)
+        question_skill_links_2 = (
+            question_models.QuestionSkillLinkModel.
+            get_question_skill_links_equidistributed_by_skill(
+                3, ['skill_id1']
+            )
+        )
+        self.assertEqual(len(question_skill_links_1), 3)
+        self.assertEqual(len(question_skill_links_2), 3)
+        self.assertNotEqual(question_skill_links_1, question_skill_links_2)
```

Comment:
Same comment applies here. Verify the contents of the response as well, not just that the two aren't the same

Reply:
Done.